src/index.css

```diff
@@ -25,3 +25,58 @@ code {
     'Courier New',
     monospace;
 }
+
+table,
+th,
+td {
+  border: 1px solid rgb(131, 131, 131);
+}
+
+.mission-table thead tr {
+  text-align: left;
+}
+
+.table-head {
+  padding: 0.5em 10px;
+}
+
+.mission-list td {
+  padding: 0.5em 10px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 18px;
+  min-width: 100px;
+}
+
+.mission-list:nth-child(even) {
+  background-color: rgb(168, 168, 168);
+}
```

Comment:
The background color used here does not correspond to the color suggested in the template which looks more like this: #f2f2f2